G/Gnuplot/build_tarballs.jl

```diff
@@ -0,0 +1,48 @@
+
+using BinaryBuilder, Pkg
+
+name = "Gnuplot"
+version = v"5.4.1"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://excellmedia.dl.sourceforge.net/project/gnuplot/gnuplot/$(version)/gnuplot-$(version).tar.gz",
+                  "6b690485567eaeb938c26936e5e0681cf70c856d273cc2c45fabf64d8bc6590e"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/gnuplot-*/
+# Make uic happy to use `libc.so` ¯\_(ツ)_/¯
+apk add g++ linux-headers samurai gnutls readline musl
```

Comment:
That's very weird, why is that needed? OK, we currently need `apk add samurai` as a workaround. But the rest seems suspicious to me.

Reply:
I don't know, there is something very wrong with `uic` in Qt: https://github.com/JuliaPackaging/Yggdrasil/blob/67f10c6e61ef7b0581f6a9a3d83dea9dbc260692/Q/Qt/build_tarballs.jl#L39.  I don't really understand why it requires `libc.so` instead of `libc.so.6` like 99.9999...% of executables.  At the moment this line isn't needed at all since we aren't using Qt

Reply:
See also https://github.com/JuliaPackaging/Yggdrasil/commit/e7967da3858f44d846d49efb66395d7228dd371d#r44397545

Reply:
gnuplot is actually available for alpine.
I was thinking about, and adding the same deps.
Doesn't work out.
I'll try adding Readline_jll